opentok/opentok.py

```diff
@@ -1639,46 +1641,88 @@ def __init__(
             app_version=app_version
         )
 
+    def mute_all(self, 
+                session_id: str, 
+                excludedStreamIds: Optional[List[str]], 
+                active: bool= True, 
+                options: dict = {}) -> requests.Response:
```

Comment:
Can we remove this parameter? We do not include this options parameter in other methods. Can the method use a private variable internally that populates the options? I don't think we should include a parameter that the developer will not use.

Reply:
Hi @jeffswartz ! Sure, I can definitely remove the options parameter in the function.